README.md

```diff
@@ -1377,13 +1401,13 @@ Sample output from [here](assets/ic_customized_workflow/result_with_path.jpg) (N
 #### Instructions
 
 1. Download the make-file.atn file from [here](assets/ic_customized_workflow/make-path.atn) (this file will be used in the Photoshop action API call)
-2. Make the first API call one to ImageCutOut service to generate intermediate result as RGBA cutout 
-3. Make the second API call to Photoshop action service to use above intermediate result as well as the make-file.atn file to generate final JPEG format result with desired PS path embedded 
-4. Open the final result with Photoshop Desktop app to check generated path in path panel 
+2. Make the first API call one to ImageCutOut service to generate intermediate result as RGBA cutout
```

Comment:
"Make the first API call to ImageCutOut Service..." 
remove the 
Make the first API call **_one_** to ImageCutOut service